src/main/java/com/redhat/coolstore/service/ShippingService.java

```diff
@@ -70,9 +74,7 @@ public double calculateShippingInsurance(ShoppingCart sc) {
     }
 
     private static double getPercentOfTotal(double value, double percentOfTotal) {
-        return BigDecimal.valueOf(value * percentOfTotal)
-                .setScale(2, RoundingMode.HALF_UP)
-                .doubleValue();
+        return value * percentOfTotal;
```

Comment:
In this case the [BigDecimal](https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html) IMHO should not be removed. Since who ever has used the type has done it for a reason.